extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -52,6 +53,19 @@ AdditionalBeanBuildItem additionalBeans() {
         return AdditionalBeanBuildItem.unremovableOf(RouterProducer.class);
     }
 
+    /**
+     * Workaround for https://github.com/quarkusio/quarkus/issues/4720 by filtering Vertx multiple instance warning in dev
+     * mode.
+     */
+    @BuildStep
+    void filterMultipleVertxInstancesWarning(LaunchModeBuildItem launchModeBuildItem,
+            BuildProducer<LogCleanupFilterBuildItem> logCleanupFilterBuildItemBuildProducer) {
+        if (launchModeBuildItem.getLaunchMode().equals(LaunchMode.DEVELOPMENT)) {
+            logCleanupFilterBuildItemBuildProducer.produce(new LogCleanupFilterBuildItem("io.vertx.core.impl",
+                    "You're already on a Vert.x context, are you sure you want to create a new Vertx instance"));
+        }
```

Comment:
@machi1990 if you tested it and it works, could you promote it? I would like to have it for tomorrow's version

Reply:
Tested it at work and the log was filtered out. Let me do another test before promoting. Thanks.

Reply:
I double checked and the log is properly filtered out. I also rebased with master so that the branch is properly up to date.